config/initializers/flipper.rb

```diff
@@ -0,0 +1,15 @@
+Flipper.configure do |config|
+  config.default do
+    adapter = if Rails.env.test?
+                Flipper::Adapters::Memory.new
+              else
+                Flipper::Adapters::ActiveRecord.new
+              end
+
+    Flipper.new(adapter)
+  end
+end
+
+Flipper.register(:admins) do |actor|
+  actor.respond_to?(:admin?) && actor.admin?
```

Comment:
I think this be reduced down to using try?

```suggestion
  actor.try(:admin?)
```

Reply:
Or even `actor&.admin?`

Reply:
Does `&` defend against a non-nil object without the method defined?

IIRC:
- `"foo".try(:non_existant_method)` returns `nil`
- `"foo".try(&:non_existant_method)` raises an exception.

I don't know which way `&` works and am too tired to Google :)

Reply:
I just copy pasted this from the wiki, actually. I will remove all `respond_to?` calls as we're dealing with the User model here.